appengine_config.py

```diff
@@ -22,8 +22,10 @@
 
 import pkg_resources
 import python_utils
+
```

Comment:
Remove the changes in this file.

Reply:
@vojtechjelinek I have made the changes.